RELEASE.md

```
@@ -0,0 +1,165 @@
+git add -A
+git commit -m "chore: bump version to 1.0.1.Final-SNAPSHOT"
+git push origin chore/bump-to-1.0.1.Final-SNAPSHOT
+```
+
+Open PR, wait for CI, and merge.
+
+### 6. Verify Deployment
+
+Check that artifacts are available on Maven Central (this can take quite some time):
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)